.gitlab/ci/.gitlab-ci.bucket-upload.yml

```diff
@@ -232,7 +232,11 @@ install-packs-in-server-master:
     - section_end "Secrets Fetch"
     - section_start "Lock Machine"
     - |
-      if [[ $TEST_UPLOAD == "true" ]]; then
+      if [[ $TEST_XDR_ENV == "true" ]]; then
+        echo "Setting CLOUD_MACHINES_TYPE to testing in test xdr-env"
+        CLOUD_MACHINES_TYPE="testing"
+        GCS_LOCKS_PATH="content-locks/locks-xsiam-ga-testing"
+      elif [[ $TEST_UPLOAD == "true" ]]; then
         echo "Setting CLOUD_MACHINES_TYPE to build in test upload-flow"
         CLOUD_MACHINES_TYPE="build"
         GCS_LOCKS_PATH="content-locks/locks-xsiam-ga"
```

Comment:
Please change your variable and existing variables to the `if: variables:` as you did it in the on-push file. 
if: $TEST_XDR_ENV == "true"
variables: ....

if $TEST_UPLOAD == "true"
variables: ....